### PR TITLE
Control fingerprinting on demand (PPTT-4318)

### PR DESCRIPTION
**Pull request by wronan**

### CODE REVIEW

**Comment by kororokke** [comment]
Master, again?

File: data_collection/api/tracking_api.yaml [hunk 1]
```diff
@@ -998,6 +998,18 @@ paths:
             type: string
             format: SemVer
           example: 3.4.1
+        - name: fp
+          in: query
+          description: |
+            Provides on-demand control of the SessionId feature (0=disabled, 1=enabled)
+
+            When this parameter is not used, processing service will default to the current value from the TrackerSettings.
```
Comment (adaqus) [inline data_collection/api/tracking_api.yaml:1006]: TrackerSettings is internal name:
```suggestion
            When this parameter is not used, processing service will default to the current value from the Privacy tab in global or app settings.
```

**Comment by wronan** [comment]
@kororokke I've changed to 18.1 but looks like it needs rebase from master...

**Comment by kororokke** [comment]
I merged master to 18.1

**Review by pmajocha — APPROVED** [review]

**Review by kororokke — APPROVED** [review]

**Pull request merged**